Beta_Integrations/integration-Vertica.yml

```diff
@@ -0,0 +1,178 @@
+description: Analytic database management software
+configuration:
+- display: Host (Vertica Server URL)
+  name: url
+  defaultvalue: myhost.example.com
+  type: 0
+  required: true
+- display: Database
```

Comment:
i would change to `Database name`, wdyt ?

Reply:
In the DB/vertica lingo the database is db name, so they are essentially the same, personally I don't see any difference, so any way goes.